test/Rules/INCOME_TAX.test.js

```diff
@@ -118,13 +118,30 @@ describe('Rules -> INCOME_TAX', () => {
         }% of networth when VARIABLE was chosen for payment `
       );
     });
+    it('should decrease player cash by config incomeTaxRate if player chooses VARIABLE, rounded to 2 decimal places', () => {
+      const startingCash = gameState.currentPlayer.cash;
+      const assets = 11;
```

Comment:
This is an arbitrary value, right ? Can we append a `// arbitrary value` if so ? Just for future-parsing